server/core/src/main/java/dev/slimevr/protocol/datafeed/DataFeedBuilder.kt

```diff
@@ -0,0 +1,429 @@
+package dev.slimevr.protocol.datafeed
+
+import com.google.flatbuffers.FlatBufferBuilder
+import dev.slimevr.guards.ServerGuards
+import dev.slimevr.tracking.processor.Bone
+import dev.slimevr.tracking.processor.skeleton.HumanSkeleton
+import dev.slimevr.tracking.processor.stayaligned.poses.RelaxedPose
+import dev.slimevr.tracking.processor.stayaligned.trackers.RestDetector
+import dev.slimevr.tracking.processor.stayaligned.trackers.StayAlignedTrackerState
+import dev.slimevr.tracking.trackers.Device
+import dev.slimevr.tracking.trackers.Tracker
+import dev.slimevr.tracking.trackers.udp.MagnetometerStatus
+import dev.slimevr.tracking.trackers.udp.UDPDevice
+import io.github.axisangles.ktmath.Quaternion
+import io.github.axisangles.ktmath.Vector3
+import solarxr_protocol.data_feed.DataFeedUpdate
+import solarxr_protocol.data_feed.device_data.DeviceData
+import solarxr_protocol.data_feed.device_data.DeviceDataMaskT
+import solarxr_protocol.data_feed.stay_aligned.StayAlignedPose
+import solarxr_protocol.data_feed.stay_aligned.StayAlignedTracker
+import solarxr_protocol.data_feed.tracker.TrackerData
+import solarxr_protocol.data_feed.tracker.TrackerDataMaskT
+import solarxr_protocol.data_feed.tracker.TrackerInfo
+import solarxr_protocol.datatypes.DeviceId
+import solarxr_protocol.datatypes.Ipv4Address
+import solarxr_protocol.datatypes.Temperature
+import solarxr_protocol.datatypes.TrackerId
+import solarxr_protocol.datatypes.hardware_info.HardwareInfo
+import solarxr_protocol.datatypes.hardware_info.HardwareStatus
+import solarxr_protocol.datatypes.math.Quat
+import solarxr_protocol.datatypes.math.Vec3f
+import java.nio.ByteBuffer
+import java.util.function.Consumer
+
+object DataFeedBuilder {
```

Comment:
for the builders files. I would move away from using object and just make functions directly without OOP. then we can import the functiuons into the different Handlers. 
Would love @ButterscotchV opinion on this.
Please note that it has already been done before on the newer builders files i did. Like Checklist.